users/views.py

```diff
@@ -3,6 +3,7 @@
 from rest_framework import permissions
 from rest_framework.response import Response
 from rest_framework.views import status
+from .serializers import CurrentUserSerializer
```

Comment:
Put a blank line above according to PEP8 (you have to split imports of the third-party packages from the imports related to the current project).